libcudacxx/include/cuda/std/__utility/integer_sequence.h

```diff
@@ -39,7 +39,19 @@ struct __integer_sequence
   using __to_tuple_indices = __tuple_indices<(_Values + _Sp)...>;
 };
 
-#ifndef _LIBCUDACXX_HAS_MAKE_INTEGER_SEQ
+#if defined(_LIBCUDACXX_HAS_MAKE_INTEGER_SEQ)
+
+template <size_t _Ep, size_t _Sp>
+using __make_indices_imp =
+  typename __make_integer_seq<__integer_sequence, size_t, _Ep - _Sp>::template __to_tuple_indices<_Sp>;
+
+#elif __has_builtin(__integer_pack) // ^^^ _LIBCUDACXX_HAS_MAKE_INTEGER_SEQ ^^^ / vvv __has_builtin(__integer_pack) vvv
```

Comment:
Suggestion: The detection seems a bit inconsistent. This PR detects support for `make_integer_seq` in the `__config` header, but support for `__integer_pack` is detected locally. I think we should move the builtin detection for `__integer_pack` into `__config` as well, like:
```c++
#  if !defined(_LIBCUDACXX_TESTING_FALLBACK_MAKE_INTEGER_SEQUENCE) && __has_builtin(__integer_pack))
#    define _LIBCUDACXX_HAS_INTEGER_PACK
#  endif
```
And then later test for the macro `#elif _LIBCUDACXX_HAS_INTEGER_PACK`.